src/controllers/expenses.controller.js

```
@@ -0,0 +1,116 @@
+const expensesService = require('../repository/expenses.repository');
+const usersService = require('../repository/users.repository');
+
+const get = async (req, res) => {
+  try {
+    const expenses = await expensesService.getAll(req.query);
+
+    res.json(expenses);
+  } catch (error) {
+    res.sendStatus(500);
```

Several catch blocks only call `res.sendStatus(500)` and swallow the error. Please add error logging (e.g., `console.error(error)`) so failures are visible during tests and debugging. Apply this consistently to all catch blocks (examples at lines 9–10, 36–37, 53–54, 71–72, 105–106).